frontend/src/v5/ui/controls/modal/formModal/formDialog.styles.ts

```diff
@@ -56,6 +56,7 @@ export const Title = styled(Typography).attrs({
 export const FormDialogContent = styled(DialogContent)`
 	margin: ${({ zeroMargin }) => (zeroMargin ? '0' : '20px 34px 43px')};
 	display: inline-flex;
+	width: fill-available;
```

Comment:
I know about webkit-fill-available, but I have never seen this property before, are you sure it is supported?

Reply:
Ha, yeah. I was intending to go back to this but forgot.